aie_runtime_lib/AIE2/vec_math.h

```diff
@@ -145,4 +145,39 @@ inline __attribute__((always_inline)) v16bfloat16 getErfBf16(v16bfloat16 in) {
   return (v16bfloat16)vec_out;
 }
 
+inline __attribute__((always_inline)) v32bfloat16 getAbs(v32bfloat16 in) {
+  aie::vector<bfloat16, 32> x = in;
+  aie::vector<bfloat16, 32> absX = aie::abs(x);
+  return (v32bfloat16)absX;
+}
+
+inline __attribute__((always_inline)) v16bfloat16 getAbs(v16bfloat16 in) {
+  aie::vector<bfloat16, 16> x = in;
+  aie::vector<bfloat16, 16> absX = aie::abs(x);
+  return (v16bfloat16)absX;
+}
+
+inline __attribute__((always_inline)) v16float getAbs(v16float in) {
+  aie::vector<float, 16> x = in;
+  aie::vector<float, 16> absX = aie::abs(x);
+  return (v16float)absX;
+}
+
+inline __attribute__((always_inline)) v16int32 getAbs(v16int32 in) {
```

Comment:
Is v32int32 supported in high-level API? If so, I think we can add the support as well, thanks. We have element-wise add/sub using v32int32 in some scenarios, which might need it. Thank you.

Reply:
In this case, we need to convert v32int32 to v32acc32 first. If we need to support more patterns, we can add more cases in the future.